packages/is-prop-valid/src/props.js

```diff
@@ -471,7 +471,8 @@ const props = {
   // preact
   for: true,
   class: true,
-  autofocus: true
+  autofocus: true,
+  video: true
```

Comment:
I believe that you have wanted to add `disablePictureInPicture` and not the `video`. Also - you should move it to the "main" list of props so it doesn look like it's preact-specific prop.

Reply:
Sure! :)